content/millennium/r4/base/individuals/related-person.md

```diff
@@ -214,15 +214,14 @@ The common [errors] and [OperationOutcomes] may be returned.
 
```

Comment:
Under create is says 'Create a new Encounter-level RelatedPerson.' but they can create a patient level too right? should we update that as well?

Reply:
There is a separate PR for Create changes that will be done in the coming days